layouts/catalog/list.html

```diff
@@ -86,6 +86,14 @@ <h1 id="works">
               {{ end }}
             </ul>
             {{ end }}
+            {{ with .additional_resources }}
```

Comment:
Thanks for working on this! I am also not a designer by any stretch... Perhaps it's obvious from my total avoidance of colour... 😅 

Anyway, it's all quite subjective, but perhaps it's a bit better above the quotes, so that it's a bit closer to the "main" link. I'd also suggest making the resources header and the resources list a tad smaller, you can use the class `f6` for this. I think the `mv0` on the header can be removed, as it doesn't seem to do anything at the moment.

Here's a sample with those tweaks in place:

<img width="935" alt="image" src="https://user-images.githubusercontent.com/279572/148246912-5affe31e-acc6-422b-a0cb-ce73c4ec705f.png">

Anyway, that's just my opinion, please let me know if you prefer your original styling. 🙂

Reply:
Yeah, I think that looks better. Let me know if you think anything else would be a good change